### ISSUE TITLE
NTR: C14:2: tetradecadienoylcarnitine high during metabolic stress in blood (Lab: chemistry: carnitine compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8561]
New term request C14:2: tetradecadienoylcarnitine high during metabolic stress in blood (segal_030514175644): Acylcarnitines

**Comment by pnrobinson** [comment]
https://hmdb.ca/metabolites/HMDB0013332

**Comment by pnrobinson** [comment]
@MickeySegal
I wonder if it would be OK to make a term Elevated circulating C14:2: tetradecadienoylcarnitine concentration and then use a modifer "triggered by metabolic stress" for the disease in question?

**Comment by MickeySegal** [comment]
This is like https://hpo.jax.org/app/browse/term/HP:0012726.  The chemistry finding is only present episodically.

**Comment by pnrobinson** [comment]
@MickeySegal 
I think it would be best to use the following new modifier, since otherwise we will get an explosion of terms.


Adding new term: 
- ID: HP:6000777
- label: Triggered by metabolic stress
- parents: [Triggered by (HP:0025204)](https://hpo.jax.org/app/browse/term/HP:0025204)
- pmid: 
- def: Applies to a sign or symptom that is provoked or brought about by a physiological or pathological response that is initiated or exacerbated by a disturbance in a metabolic processes, such as fasting, hypoxia, infectious disease, accumulation of metabolic byproducts, or an imbalance in energy production and expenditure.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8561
- synonyms:

**Comment by MickeySegal** [comment]
This new term seems vague and unnecessary.  We already have https://hpo.jax.org/browse/term/HP:0025212 and https://hpo.jax.org/browse/term/HP:0025215 which define types of metabolic stress.  The new term is vague because it doesn't make clear what the metabolic stress is.  

We need something specific, however, for the tetradecadienoylcarnitine elevation, as we did for https://hpo.jax.org/browse/term/HP:6000476.
